src/shared/components/TimeSeries.tsx

```diff
@@ -261,7 +262,7 @@ class TimeSeries extends Component<Props, State> {
 
       let statuses = [] as StatusRow[][]
       if (check) {
-        const extern = buildVarsOption(vars)
+        const extern = buildUsedVarsOption(queries as string[], variables)
```

Comment:
Something is fishy here. The queries shouldn't be typecast, especially since the type is set at the top as `queries: DashboardQuery[]`